p2p/pex/addrbook.go

```diff
@@ -487,19 +493,19 @@ func (a *addrBook) getBucket(bucketType byte, bucketIdx int) map[string]*knownAd
 
 // Adds ka to new bucket. Returns false if it couldn't do it cuz buckets full.
 // NOTE: currently it always returns true.
-func (a *addrBook) addToNewBucket(ka *knownAddress, bucketIdx int) bool {
+func (a *addrBook) addToNewBucket(ka *knownAddress, bucketIdx int) {
```

Comment:
We also need to modify `addToOldBucket` then for consistency, right?

Reply:
we actually use the response from addToOldBucket ... but this code could all use a clean up ...